app/workers/retirement_worker.rb

```diff
@@ -18,4 +19,13 @@ def deactivate_workflow!(workflow)
       Workflow.where(id: workflow.id).update_all(active: false)
     end
   end
+
+  def push_counters_to_event_stream(workflow)
```

Comment:
no event_id here, you happy to use the defaults from the es writer [here](https://github.com/zooniverse/zoo-event-stats/blob/dcdc0ec411d5924b32965e43a7b5f48e445c6334/lib/output/elasticsearch_writer.rb#L48)?